jekyll-export-cli.php

```diff
@@ -16,12 +16,54 @@
  *
  * Must be run in the wordpress-to-jekyll-exporter/ directory.
  */
-require '../../../wp-load.php';
+
+// Uncomment for extra replace options
+
+$wordpress_path = "../../../";
+//$wordpress_path = "/home/wordpress/";
```

Comment:
We'll want to ensure any changes are generalized, and not specific to a particular setup, even if commented out.

Reply:
This was intended to be generalized to provide also an example with absolute path.  I rephrased as documentation.